java/number-adder/src/test/java/pl/edu/icm/rjava_project_example/AdvancedAdderTestCase.java

```diff
@@ -0,0 +1,55 @@
+package pl.edu.icm.rjava_project_example;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+/**
+ * @author Mateusz Kobos
+ */
+public class AdvancedAdderTestCase {
```

Comment:
Just my convention. I guess I got used to it when in older versions of JUnit, your test class had to inherit from `TestClase` class.